Cargo.toml

```diff
@@ -13,16 +13,16 @@ bitflags = "2.0"
 byteorder = "1.3.2"
 cfg-if = "1.0"
 crash-context = "0.6"
-memoffset = "0.8"
-minidump-common = "0.16.0"
+memoffset = "0.9"
+minidump-common = "0.17.0"
 scroll = "0.11"
 tempfile = "3.1.0"
 thiserror = "1.0.21"
 
 [target.'cfg(unix)'.dependencies]
 libc = "0.2.74"
-goblin = "0.6"
-memmap2 = "0.5"
+goblin = "0.7"
+memmap2 = "0.7"
```

Comment:
rust-minidump seems to be pulling in 0.5.7 and that's part of the reason for the failures on macOS, I'm going to update it

Reply:
Or we do the opposite and go back to 0.5.x until they update first.

Reply:
Yah that's what I opted for, easiest.